Makefile

```diff
@@ -44,7 +44,7 @@ TRIVY := trivy
 
 # Image URL to use all building/pushing image targets
 REGISTRY ?= docker.io
-IMAGE_NAME ?= secrets-store-sync-controller
+IMAGE_NAME ?= sync-controller
```

Comment:
Can we call the image controller? It will be pulled as `us-central1-docker.pkg.dev/k8s-staging-images/secrets-store-sync/controller:latest` and `registry.k8s.io/secrets-store-sync/controller:latest`